gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/SweRunner.java

```diff
@@ -16,25 +18,38 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
 
+import java.time.OffsetDateTime;
+
 /**
  * @author Theo Pascoli {@literal <theo.pascoli at rte-france.com>}
  */
 @Service
 public class SweRunner {
     private static final Logger LOGGER = LoggerFactory.getLogger(SweRunner.class);
+
+    public static final String CRAC_CIM_CRAC_CREATION_PARAMETERS_PT_ES_JSON = "/crac/CimCracCreationParameters_PT-ES.json";
+    public static final String CRAC_CIM_CRAC_CREATION_PARAMETERS_FR_ES_JSON = "/crac/CimCracCreationParameters_FR-ES.json";
+
     private final NetworkService networkImporter;
     private final FileImporter fileImporter;
+    private final FileExporter fileExporter;
 
-    public SweRunner(NetworkService networkImporter, FileImporter fileImporter) {
+    public SweRunner(NetworkService networkImporter, FileImporter fileImporter, FileExporter fileExporter) {
         this.networkImporter = networkImporter;
         this.fileImporter = fileImporter;
+        this.fileExporter = fileExporter;
     }
 
     @Threadable
     public SweResponse run(SweRequest sweRequest) {
         LOGGER.info("Request received for timestamp {}", sweRequest.getTargetProcessDateTime());
         Network network = networkImporter.importNetwork(sweRequest);
-        Crac crac = fileImporter.importCimCracFromUrlWithNetwork(sweRequest, network);
+        CimCrac cimCrac = fileImporter.importCimCrac(sweRequest);
+        OffsetDateTime targetProcessDateTime = sweRequest.getTargetProcessDateTime();
+        Crac cracEsPt = fileImporter.importCracFromCimCracAndNetwork(cimCrac, targetProcessDateTime, network, CRAC_CIM_CRAC_CREATION_PARAMETERS_PT_ES_JSON);
+        Crac cracFrEs = fileImporter.importCracFromCimCracAndNetwork(cimCrac, targetProcessDateTime, network, CRAC_CIM_CRAC_CREATION_PARAMETERS_FR_ES_JSON);
+        String jsonPathEsPt = fileExporter.saveCracInJsonFormat(cracEsPt, "cracEsPt.json", targetProcessDateTime, ProcessType.D2CC);
```

Comment:
The constant "cracEsPt.json" could also be extracted to make it easier to modify in the future